[BE] Clarify comment to not revert when command has been edited

janeyx99 opened the pull request:
This is mostly a nit. I was a bit confused when I saw 
<img width="1032" height="183" alt="image" src="https://github.com/user-attachments/assets/7a18f167-78c1-4c33-ba6f-3588914c642e" />
in https://github.com/pytorch/pytorch/pull/159172

So I decided I should clean up this message a bit.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #159495

janeyx99 committed:
[BE] Clarify comment to not revert when command has been edited

[ghstack-poisoned]

clee2000 reviewed (approved):
maybe revert comment? instead of command

janeyx99 committed:
Update on "[BE] Clarify comment to not revert when command has been edited"

This is mostly a nit. I was a bit confused when I saw 
<img width="1032" height="183" alt="image" src="https://github.com/user-attachments/assets/7a18f167-78c1-4c33-ba6f-3588914c642e" />
in https://github.com/pytorch/pytorch/pull/159172

So I decided I should clean up this message a bit.




[ghstack-poisoned]

malfet commented:
@pytorchbot merge -f "Looks like I just like force merges (but also, lint is green)"
